src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/MainErrorReportViewControllers/TopErrorReportViewModel.swift

```diff
@@ -0,0 +1,85 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import OpenCombine
+
+class TopErrorReportViewModel {
+	
+	// MARK: - Init
+	
+	init(didPressHistoryCell: @escaping () -> Void) {
+		self.didPressHistoryCell = didPressHistoryCell
+	}
+	
+	// MARK: - Internal
+	
+	func updateViewModel(isHistorySectionIncluded: Bool = false) {
+		var dynamic = DynamicTableViewModel([])
+		dynamic.add(
+			.section(cells: [
+				.body(text: AppStrings.ErrorReport.description1),
+				.link(text: AppStrings.ErrorReport.faq, url: URL(string: "https://example.com"), accessibilityIdentifier: AccessibilityIdentifiers.ErrorReport.faq) // TO DO: get correct link!
+			])
+		)
+		dynamic.add(
+			.section(cells: [
+				.acknowledgement(
+					title: NSAttributedString(string: AppStrings.ErrorReport.Legal.dataPrivacy_Headline),
+					description: nil,
+					bulletPoints: [
+						NSMutableAttributedString(string: AppStrings.ErrorReport.Legal.dataPrivacy_Bullet1),
+						NSMutableAttributedString(string: AppStrings.ErrorReport.Legal.dataPrivacy_Bullet2),
+						NSMutableAttributedString(string: AppStrings.ErrorReport.Legal.dataPrivacy_Bullet3),
+						NSMutableAttributedString(string: AppStrings.ErrorReport.Legal.dataPrivacy_Bullet4),
+						NSMutableAttributedString(string: AppStrings.ErrorReport.Legal.dataPrivacy_Bullet5)
+					],
+					accessibilityIdentifier: "TODO ACCESSABILITY IDENTIFIER")
```

Comment:
Just create one! ;)

Reply:
I'm doing so now in the UITesting branch